### ISSUE TITLE
Missing closing parenthesis

### ISSUE THREAD

**Issue by gvwilson** [rangle/angular-training-slides#531]
Missing closing paren on "set the summary attribute to null"

**Comment by neelghosh** [comment]
Added - pr 531 - committed but not pushed (getting permission error again)
